Panic when 404 pull request url

vot takaya oshibka valitsa:

> work:~ $ ash treetrunks/2 review 
12:02:24.63 [WARN] Stash returned error code: %d 404
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x48 pc=0x40f108]
> 
goroutine 1 [running]:
panic(0x82f700, 0xc82000e150)
        /usr/lib/go/src/runtime/panic.go:481 +0x3e6
main.review(0xc8200e7360, 0xc8203d3560, 0x2, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
        /home/a.baranov/.go/src/github.com/seletskiy/ash/main.go:646 +0x1c8
main.reviewMode(0xc82043aa80, 0xc8200e72e0, 0x7fff28e57239, 0xa, 0xc8203d34a0, 0x2)
        /home/a.baranov/.go/src/github.com/seletskiy/ash/main.go:319 +0xcf7
main.main()
        /home/a.baranov/.go/src/github.com/seletskiy/ash/main.go:169 +0xe4bw

--- a/api.go
+++ b/api.go
@@ -113,10 +113,10 @@ func (api Api) doRequest(
 	}
 
 	if err := checkErrorStatus(resp); err != nil {
-		logger.Warning("Stash returned error code: %d", resp.Raw.StatusCode)
+		logger.Warningf("Stash returned error code: %d", resp.Raw.StatusCode)
 		return err
 	} else {
-		logger.Debug("Stash returned status code: %d", resp.Raw.StatusCode)
+		logger.Debugf("Stash returned status code: %d", resp.Raw.StatusCode)
 	}
 
 	return nil

--- a/main.go
+++ b/main.go
@@ -643,6 +643,11 @@ func review(
 			review, err = pr.GetReview(path, ignoreWhitespaces)
 		}
 
+		if review == nil {
+			fmt.Println("Pull request not found")
+			os.Exit(1)
+		}
+
 		if len(review.changeset.Diffs) == 0 {
 			fmt.Println("Specified file is not found in pull request.")
 			os.Exit(1)

--- a/main.go
+++ b/main.go
@@ -644,7 +644,7 @@ func review(
 		}
 
 		if review == nil {
-			fmt.Println("Pull request not found")
+			fmt.Fprintln(os.Stderr, "Pull request not found.")
 			os.Exit(1)
 		}
 

